docs/install/_index.md

```diff
@@ -1,8 +1,21 @@
 ---
 title: "Installing Knative"
-linkTitle: ""
 weight: 05
 type: "docs"
+showlandingtoc: "false"
 ---
 
-{{% readfile file="README.md" %}}
+You can install the Serving component, Eventing component, or both on your cluster by using one of the following deployment options:
+
+- Using a [YAML-based installation](./any-kubernetes-cluster).
+- Using the [Knative Operator](./knative-with-operators).
+- Following the documentation for vendor managed [Knative offerings](../knative-offerings).
+
+You can also [upgrade an existing Knative installation](./upgrade-installation).
+
+**NOTE:** Currently Knative instructions are only provided for Mac and Linux environments.
```

Comment:
I read this as being a bit stronger than it needs to be (or was before). The instructions assume Mac or Linux, but should work with pretty minor modifications on windows. WDYT about something like:

```suggestion
**NOTE:** Currently the Knative install instructions assume you are running Mac or Linux with a bash shell.
```